lib/roo_on_rails/routemaster/publisher.rb

```diff
@@ -33,6 +33,10 @@ def url
         raise NotImplementedError
       end
 
+      def async?
+        false
```

Comment:
Also, making the default `true` means you'd _have_ do have Sidekiq setup. I think we'd rather not force users to use all the features in `roo_on_rails`?